CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## 1.79.0
+* Feature - Add support for finding EBS devices on Xen instances [#3971](https://github.com/aws/amazon-ecs-agent/pull/3971)
+* Feature - Add network builder and platform APIs [#3939](https://github.com/aws/amazon-ecs-agent/pull/3939)
+* Enhancement - Load managed daemon images in background to speed up startup time [#3984](https://github.com/aws/amazon-ecs-agent/pull/3984)
+* Enhancement - ECS client API update [#3974](https://github.com/aws/amazon-ecs-agent/pull/3974)
+* Enhancement - Move utils function Uint16SliceToStringSlice to shared library [#3981](https://github.com/aws/amazon-ecs-agent/pull/3981)
+* Enhancement - Move instance credentials and credential provider to shared library [#3980](https://github.com/aws/amazon-ecs-agent/pull/3980)
```

Comment:
these seem to align with others in the "code quality improvement" category

Reply:
Should be updated